docs/docs/charts/radar.mdx

```diff
@@ -92,6 +92,7 @@ The radar chart allows a number of properties to be specified for each dataset.
 | [`hoverBorderJoinStyle`](#line-styling) | `string` | Yes | - | `undefined`
 | [`hoverBorderWidth`](#line-styling) | `number` | Yes | - | `undefined`
 | [`clip`](#general) | `number`\|`object` | - | - | `undefined`
+| [`data`](#data-structure) | `number[]` | - | - | **required**
```

Comment:
Would have thought other formats worked, but I was wrong. The scale does not implement the parsing methods required.